lib/util/esWrapper.js

```diff
@@ -25,7 +25,7 @@ const
   Bluebird = require('bluebird'),
```

Comment:
Please revert all changes made to this file. You have merged the `1.x` version of this file in the `2.x` branch